app/models/post.rb

```diff
@@ -3,6 +3,9 @@ class Post < ApplicationRecord
   has_many :comments
   has_many :likes
   after_save :update_posts_counter
+  validates :title, presence: true
```

Comment:
- [ ] Please add validation for the post's title so that it does not exceed 250 characters. As per [this](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_validations_model_spec.md#:~:text=Title%20must%20not%20exceed%20250%20characters.) requirement.